other/maximum_subsequence.py

```diff
@@ -0,0 +1,31 @@
+from collections.abc import Sequence
+
+
+def max_subsequence_sum(nums: Sequence[int]) -> int:
+    """Return the maximum possible sum amongst all non - empty subsequences.
+
+    Raises:
+      ValueError: when nums is empty.
+
+    >>> max_subsequence_sum([1,2,3,4,-2])
+    10
+    >>> max_subsequence_sum([-2, -3, -1, -4, -6])
+    -1
+    """
+    if not nums:
+        raise ValueError("Input sequence should not be empty")
```

Comment:
This is kind of implied, but if you want to have this check, create a doctest that fails with it
ie 
```py
>>> max_subsequence_sum()
Traceback (most recent call last):
    ....
ValueError: Input sequence should not be empty
```

Reply:
Ok, I will do so.